crates/cargo-wdk/src/actions/build/mod.rs

```diff
@@ -429,7 +411,7 @@ impl<'a> BuildAction<'a> {
         )
         .run()?;
 
-        info!("Processing completed for package: {}", package_name);
+        info!("Finished building {package_name}");
```

Comment:
I wonder if "building" is the right thing here since its also done all the packaging steps. For every other step, we assuming that after its done, it goes to the next step. "building" is already the first step that was completed

Reply:
compilation vs build is probably a good distinction. im fine with you addressing that as part of your imminent refactors